pkgs/misc/urbit/default.nix

```diff
@@ -1,14 +1,13 @@
-{ stdenv, fetchgit, gcc, gmp, libsigsegv, openssl, automake, autoconf, ragel,
+{ stdenv, fetchzip, gcc, gmp, libsigsegv, openssl, automake, autoconf, ragel,
   cmake, re2c, libtool, ncurses, perl, zlib, python }:
 
 stdenv.mkDerivation rec {
 
   name = "urbit-${version}";
   version = "2016-06-02";
 
-  src = fetchgit {
-    url = "https://github.com/urbit/urbit.git";
-    rev = "8c113559872e4a97bce3f3ee5b370ad9545c7459";
+  src = fetchzip {
```

Comment:
If you prefer, `fetchFromGitHub` is a convenience wrapper which expands to what you've got here.